fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -419,6 +420,9 @@ private void updateMeta(Set<String> mvRefreshedPartitions,
                 throw new DmlException(
                         "update meta failed. materialized view:" + materializedView.getName() + " not exist");
             }
+            // invalid mv's plan cache because plan may change
+            CachingMvPlanContextBuilder.getInstance().invalidateFromCache(materializedView);
```

Comment:
Why invalid this? Only invalid when necccesary, otherwise the cache maybe overhead if the refresh is too quick.